Client/Frontend/Library/BookmarksPanel.swift

```diff
@@ -197,6 +228,18 @@ class BookmarksPanel: SiteTableViewController, LibraryPanel {
         doDelete()
     }
 
+    func didAddBookmarkNode() {
+        flashLastRowOnNextReload = true
+    }
+
+    func flashRow(at indexPath: IndexPath) {
+        tableView.selectRow(at: indexPath, animated: true, scrollPosition: .middle)
```

Comment:
Due to the async behaviour, this func should verify the index path exists before trying to perform an operation on it.
i.e. `indexPath.section < numberOfSections && indexPath.row < numberOfRows(inSection: indexPath.section)`